en/docs/includes/obtain-api-key.md

```diff
@@ -0,0 +1,7 @@
+After the APIs are exposed via Choreo Connect, you can invoke an API with a valid token(JWT or opaque access token) or an API key.  Let's use Choreo Connect's API key endpoint to obtain an API key in order to access the API.
```

Comment:
I believe we removed this file since it was not used anywhere. Has this changed?

Reply:
This is used in https://github.com/wso2/docs-apim/blob/master/en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/deploy-api/deploy-api-to-choreo-connect.md